packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardWithValidation.js

```diff
@@ -122,7 +122,7 @@ class LoginCardWithValidation extends React.Component {
   handleOnInputErrors = () => {
     const { usernameField, passwordField } = this.state;
     if (usernameField.value) {
-      !this.isUserNameValid() && this.handleOnInvalidUsername();
+      !this.isEmailValid() && this.handleOnInvalidUsername();
```

Comment:
why? do you always assume that username is an email?

Reply:
Inside this function I validate only it's an email, else return true